docs/1.0.md

```diff
@@ -0,0 +1,30 @@
+
+# RSS Namespace Extension for Podcast
+
+A wholistic RSS namespace for podcasting that is meant to synthesize the fragmented world of podcast namespaces. The broad goal is to create one namespace to rule them all, that is easily extensible, community controlled/authored and addresses the needs of the independent podcast industry now and in the future. The large podcast platforms have shown virtually no interest in extending their namespaces for new functionality in many years. Our hope is that this namespace will become the framework that the independent podcast community needs to deliver new functionality to apps and aggregators.
+# Podcast Tags
+
+## Transcript
+`<podcast:transcript>`
+This tag is used to link to a transcript or closed captions file. Multiple tags can be present for multiple transcript formats.
+
+#### Parent
+`<item>`
+
+#### Attributes
+**url:** URL of the podcast transcript
+
+**type:** Mime type of the file such
+as `text/plain`, `text/html`, `application/srt`, `application/json`
+
+**language (optional):** The language of the linked transcript. If there is no language attribute given, the linked file is assumed to be the same language that is
+specified by the RSS `<language>` element.
+
+**rel (optional):** If the rel="captions" attribute is present, the linked file is considered to be a closed captions file, regardless of what the mime type is.  In that scenario, time codes are assumed to be present in the file in some capacity.
```

Comment:
I'm still not totally clear on the use of the `rel` tag. The way it reads, I'm not sure if you should or should not provide it when you use a mime type of something like `srt`.